pom.xml

```diff
@@ -16,6 +17,19 @@
             https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml
         </maven.checkstyle.plugin.configLocation>
     </properties>
+    <dependencies>
+        <dependency>
```

Comment:
should we add to dependency plugin? Does it work without dependency?

Reply:
Yes, plugin without dependensy didn't work for me